ospd_openvas/daemon.py

```diff
@@ -355,6 +361,43 @@ def feed_is_outdated(self, current_feed):
             return True
         return False
 
+    def feed_locked(self):
+        """ Check if there is an already lock file set for the feed. """
+        if self.feed_lock_file.is_file():
+            logger.info(
+                "A feed update process is running. Trying again later..."
+            )
+            return True
+
+        return False
+
+    def create_feed_lock_file(self):
+        """ Create a lock file.
+            Return: True in success, False otherwise.
+        """
+        if self.feed_locked():
+            return False
+        else:
+            try:
+                with open(str(self.feed_lock_file), 'w') as f:
```

Comment:
```suggestion
                with self.feed_lock_file.open('w') as f:
```
Details at https://docs.python.org/3/library/pathlib.html#pathlib.Path.open